### PR TITLE
Update cross-entropy-softmax.ipynb

### PR DESCRIPTION
**Pull request by frechette-alex**
I think you lost a negative sign in there.

### CODE REVIEW

**Comment by peterroelants** [comment]
Thank for flagging!
